src/index.ts

```diff
@@ -14,7 +14,8 @@ export default function lambdaHandler(app: Application): AwsLambdaHandler {
       awsRequest.isBase64Encoded ? Buffer.from(awsRequest.body, 'base64') : awsRequest.body,
     );
     const response = new MemoryResponse();
-    const context = new Context(request, response);
+    const context = new Context(request, response) as AwsContext;
+    context.aws = { context, event: awsRequest };
```

Comment:
I don't think it's needed to nest the context, I would just want to add the lambda specific objects to `context.aws`